quickstarts/eap-camel-amq-template.json

```diff
@@ -181,6 +181,20 @@
       "name": "CPU_LIMIT",
       "value": "1.0",
       "required": true
+    },
+    {
+      "name": "BUILD_MEMORY_REQUEST",
+      "displayName": "Build Memory request",
+      "value": "0.7G",
+      "required": true,
+      "description": "The amount of memory to requests for builds."
```

Comment:
Minor thing about the BUILD_MEMORY_REQUEST param description - I think we should change 'requests' to 'request'.